### PR TITLE
new recipe - heimer-1.10.0

### PR DESCRIPTION
**Pull request by mazbrili**
new recipe ..

### CODE REVIEW

File: app-office/heimer/heimer-1.10.0.recipe [hunk 1]
```diff
@@ -0,0 +1,79 @@
+SUMMARY="A Qt Mind map application"
+DESCRIPTION="Desktop application for creating mind maps and other suitable diagrams.\
```
Comment (diversys) [inline app-office/heimer/heimer-1.10.0.recipe]: Space is missing after the period.

File: app-office/heimer/heimer-1.10.0.recipe [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+REVISION="1"
+SOURCE_URI="https://github.com/juzzlin/Heimer/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="d6850ab08825c2e5cb0278782a24a4fde8c16059bb6c2a23fbdd63f2952db0ff"
+SOURCE_FILENAME="Heimer-$portVersion.tar.gz"
+SOURCE_DIR="Heimer-$portVersion"
+ADDITIONAL_FILES="heimer.rdef.in"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+ SECONDARY_ARCHITECTURES="x86"
```
Comment (diversys) [inline app-office/heimer/heimer-1.10.0.recipe]: Ident with a tab, not space.

File: app-office/heimer/heimer-1.10.0.recipe [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+BUILD()
+{
+	qmake
+	make $jobArgs
+}
+
+INSTALL()
+{
+	install -d "$appsDir"
```
Comment (diversys) [inline app-office/heimer/heimer-1.10.0.recipe]: You can omit double quotes here and below.

Comment (mazbrili) [reply app-office/heimer/heimer-1.10.0.recipe]: ok done

File: app-office/heimer/heimer-1.10.0.recipe (hunk 2 above)
Comment (mazbrili) [reply app-office/heimer/heimer-1.10.0.recipe]: thanks to remind. done

File: app-office/heimer/heimer-1.10.0.recipe (hunk 1 above)
Comment (mazbrili) [reply app-office/heimer/heimer-1.10.0.recipe]: oops.. sorry. done

**Comment by mazbrili** [comment]
thanks for your help @diversys

**Pull request merged**